setup.py

```diff
@@ -1,23 +1,18 @@
 #!/usr/bin/env python
-"""NumPy: array processing for numbers, strings, records, and objects.
+""" NumPy is the fundamental package for array computing with Python.
+It contains among other things:
```

Comment:
nit: comma needed? Or change to "It provides:" and then at the end
```
- and much more
```

Reply:
thanks, done